fileserver/routes.py

```diff
@@ -345,24 +353,39 @@ def get_session_version():
 
         updated = row[0]
 
-        # Fetch the latest release version
-        cur.execute(
-            """
-            SELECT id, version, name, notes from release_versions
-            WHERE proj_name = %s ORDER BY version_code DESC""",
-            (project,),
-        )
+        if release_channel == "alpha":
+            # Fetch the latest alpha release version
+            cur.execute(
+                """
+                SELECT id, version, name, notes, alpharelease from alpharelease_versions
+                WHERE proj_name = %s ORDER BY version_code DESC""",
+                (project,),
+            )
+        else:
+            # Fetch the latest release version
+            cur.execute(
+                """
+                SELECT id, version, name, notes, alpharelease from release_versions
+                WHERE proj_name = %s ORDER BY version_code DESC""",
+                (project,),
+            )
```

Comment:
Perhaps just use an f-string on the view name here to avoid most of the duplication (unless I'm missing some other difference?):
```suggestion
        # Fetch the latest release (or alpha release) version
        cur.execute(
            f"""
            SELECT id, version, name, notes, alpharelease
            FROM {'alpha' if release_channel == "alpha" else ''}release_versions
            WHERE proj_name = %s ORDER BY version_code DESC""",
            (project,),
        )
```

Reply:
You haven't missed anything and are correct. I have fixed it. 👍